### DISCUSSION TITLE
Will it still take effect?

### DISCUSSION THREAD

**Discussion by Rovermat** [RikkaApps/Shizuku#563] [Q&A]
I am so so so sorry to ask this stupid question, but I am a little bit confused for it because I just started using Shizuku and App ops without root today. I am currently using samsung s23u and the Auto Blocker (Samsung built-in feature) has to be turned off to turn on the wireless debugging (If auto blocker is on, the wireless debugging in developer options will be greyed out), to use Shizuku and app ops. And I feel a little insecure that auto blocker is off as it helps to secure my phone.

So I read your website (https://appops.rikka.app/guide/working_mode/shizuku/) and I saw "If you use adb, you need to use adb to start Shizuku every time your device boot (**but the appops settings always take effect**)". Is that mean if even I restart the phone or I turn the wireless debugging off (aka. turn on auto blocker), the appops settings that I have set (eg. Chrome ---- app permissions ---- Camera ---- Ignore ) are still take effect?
![Appops](https://github.com/user-attachments/assets/c3035500-f4af-49c3-8662-adee07c75d79)

**Comment by shaggyskunk** [comment]
I don't use AutoBlocker on my S23 Ultra, and no issues with Security at all.
